BenadrylAutomation/src/main/java/com/jnj/pageobjects/SafetyInformationPage.java

```diff
@@ -0,0 +1,140 @@
+/**
+ * @author Vaibhav Nagvekar
+ * @date 15-Mar-24
+ */
+
+package com.jnj.pageobjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import com.jnj.actions.Action;
+import com.jnj.base.BaseClass;
+
+public class SafetyInformationPage extends BaseClass {
+	Actions actions;
+
+	/**
+	 * Constructor
+	 */
+	public SafetyInformationPage() {
+		PageFactory.initElements(driver, this);
+		actions = new Actions(driver);
+	}
+
+	/**
+	 * Locators
+	 */
+	@FindBy(xpath = "//h1[@id='content-main']")
+	WebElement bannerTitle;
```

Comment:
@vaibhavNagvekarSJI define the locators as private

Reply:
fixed

Reply:
Verified @vaibhavNagvekarSJI